python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -268,16 +269,18 @@ def __from_edgelist(
             dst_col_name = self.renumber_map.renumbered_dst_col_name
 
         ddf = self.edgelist.edgelist_df
-
+        _client = default_client()
```

Comment:
Thanks, Will add more comments.

Reply:
Also, this code seems like it can probably be reused in other places where we want to divide a dataframe into 1 partition per worker prior to calling PLC.  I'm not saying we update every algorithm right now, but maybe we add this as a utility function that could possible replace `get_distributed_data()` at some point?